W-14476576 DC sync not req duke (includes W-14476577,  W-14476579,  W-14776338) (#263)
mulesoft/docs-code-builder: modules/ROOT/pages/des-create-api-fragments.adoc

Reviewer: Business Group has to be removed from API Fragment form

Author: @carobazan This file was removed. I'm not sure what content this comment refers to.

See the xref:tutorials.adoc[] for details.
* Have some familiarity with xref:access-management::business-groups.adoc[business groups].
+

> When you publish your API spec to Exchange from Anypoint Code Builder, the IDE requests the name of the business group.

Author: ```suggestion
When you publish your API fragment to Exchange from Anypoint Code Builder, the IDE requests the name of the business group. See xref:access-management::business-groups.adoc[].
```

See the xref:tutorials.adoc[] for details.
* Have some familiarity with xref:access-management::business-groups.adoc[business groups].
+

> //TODO: VERIFY

Author: ```suggestion
```

verified in build 29

> //TODO: VERIFY

Author: todo

Author: verified
```suggestion
```

As you enter elements, use xref:start-discover-ui.adoc#use-autocomplete[auto-complete] (Ctrl+Space) to display available options within the context.
+
If you created an OAS 3.0 (YAML) fragment project, you can replace the initial API fragment with the <<example-oas-api-frag,example>> code to test adding fragments to a spec.

> //TODO (FEB): SHOULD WE HAVE AN OPTIONAL STEP FOR SYNCING WITH THEIR SCM? 

Author: ```suggestion
```

As you enter elements, use xref:start-discover-ui.adoc#use-autocomplete[auto-complete] (Ctrl+Space) to display available options within the context.
+
If you created an OAS 3.0 (YAML) fragment project, you can replace the initial API fragment with the <<example-oas-api-frag,example>> code to test adding fragments to a spec.

> //TODO (FEB): SHOULD WE HAVE AN OPTIONAL STEP FOR SYNCING WITH THEIR SCM? 

Author: todo

Author: ```suggestion
```

Author: it's on the landing page and prob added to an existing spec
